mount/mount_windows.go

```diff
@@ -166,6 +185,18 @@ func removeSMBMapping(remotepath string) (string, error) {
 func (mounter *Mounter) Unmount(target string) error {
 	klog.V(4).Infof("azureMount: Unmount target (%q)", target)
```

Comment:
This can be done separately, but can we clean up these "azure" strings from the logs, since this is supposed to be a provider-independent library?

Reply:
removed it.